test/metadata_service/get_metadata_service_endpoint.spec.js

```diff
@@ -0,0 +1,57 @@
+var AWS, helpers, getMetadataServiceEndpoint, Endpoint, EndpointMode, ENDPOINT_CONFIG_OPTIONS, ENDPOINT_MODE_CONFIG_OPTIONS;
```

Comment:
Is this required? Other files was transfiled from CoffeeScript, so was not ideal. Since this is hand-written, it'd better to write it normally.

Reply:
It's not required. It was the first test file I wrote and just copied from `metadata_service.spec.js`

Line removed in [`caa90c4` (#3853)](https://github.com/aws/aws-sdk-js/pull/3853/commits/caa90c48bd394092f207d4c2be63c902261255df)